app/backplane/radio_module/src/c_gnss_tenant.cpp

```diff
@@ -62,7 +62,19 @@ void CGnssTenant::Startup() {
 }
 
 void CGnssTenant::PostStartup() {
+    NTypes::GnssBroadcastPacket gnssBroadcastPayload {
+        .Coordinates = gnssData.Coordinates,
+        .SatelliteCount = gnssData.Info.SatelliteCount,
+        .FixStatus = gnssData.Info.FixStatus,
+        .FixQuality = gnssData.Info.FixQuality
+    };
 
+    NTypes::LoRaBroadcastData loraBroadcastPacket {
+        .Port = NNetworkDefs::RADIO_MODULE_GNSS_DATA_PORT,
+        .Size = sizeof(NTypes::GnssBroadcastPacket)
+    };
+    memcpy(loraBroadcastPacket.Payload, &gnssBroadcastPayload, sizeof(NTypes::GnssBroadcastPacket));
```

Comment:
are you clobbering Port and Size with that memcpy?

Reply:
How so? GnssBroadcastPacket is 16 bytes. The buffer stored is 254 bytes

Reply:
Btw this does pretty much the exact same as the below code. I did decide to break it out into a function and just call it in both now though

Reply:
oh yea i was reading lorabroadcastPacket not loraBroadcastPacket.Payload so i was thinking it was overwriting the Port and Size.